src/apps/assets/avatars/components/EditAvatar/index.tsx

```diff
@@ -0,0 +1,171 @@
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
+import { useParams, useRouter } from '@tanstack/react-router';
+import {
+  Alert,
+  Button,
+  Card,
+  Descriptions,
+  Form,
+  Image,
+  Input,
+  Spin,
+  Typography,
+  message,
+} from 'antd';
+import dayjs from 'dayjs';
+import { Controller, useForm } from 'react-hook-form';
+import { z } from 'zod';
+
+import { avatarQueries } from '@/shared/service/query/avatar';
+
+const schema = z.object({
+  name: z.string().min(1, '이름을 입력해주세요.'),
+  description: z.string().optional(),
+});
+
+type FormValues = z.infer<typeof schema>;
+
+export default function EditAvatar() {
+  const { id: avatarId } = useParams({
+    from: '/(menus)/assets/avatars/$id/edit/',
+  });
+  const router = useRouter();
+  const queryClient = useQueryClient();
+
+  const {
+    data: avatar,
+    isLoading,
+    isError,
+  } = useQuery({
+    ...avatarQueries.getSingle(avatarId!),
+    enabled: !!avatarId,
+  });
+
+  const { mutate, isPending } = useMutation({
```

Comment:
```suggestion
  const { mutate: updateAvatar, isPending } = useMutation({
```
취향일수 있지만 mutate가 동작을 명확하게 나타내지 못한다고 생각이 들어서 이름 좀만 더 구체적으로 작성해주면 좋을거같아요

Reply:
변수명 주의해서 작성하겠습니다!